solidstart-tanstackquery-tailwind-modules/src/routes/orgs/[login].tsx

```diff
@@ -61,21 +68,31 @@ export default function OrgProfile() {
             </Switch>
           </div>
           <div class="col-span-12 md:col-span-8 xl:col-span-12">
-            {/* TODO:  <TabNav /> goes here with class="border-none */}
+            <ProfileNav class="border-none" pathname={location.pathname} />
           </div>
         </div>
       </div>
       <div class="max-w-screen-lg mx-auto py-8 px-2">
         <div class="grid grid-cols-12 gap-8">
           <div class="col-span-12 md:col-span-8 xl:col-span-12">
-            {/* TODO:  <TabNav /> goes here with class="border-none */}
+            <ProfileNav
+              class="border-none md:hidden"
+              pathname={location.pathname}
+            />
             <RepoFilter
               languages={repoLanguages()}
               filteredRepoCount={repos().length}
             />
             <For each={repos()}>
               {(props) => <RepoCard {...props} isProfilePage />}
             </For>
+
+            <Show when={pageInfo()?.hasNextPage || pageInfo()?.hasPreviousPage}>
+              <Pagination
+                pageInfo={pageInfo() as PageInfo}
```

Comment:
Since PageInfo could be `undefined`, I think we could remove this cast and modify the Pagination component (`src/components/Pagination/Pagination.tsx`) as follow:


```ts
interface IProps {
  pageInfo: PageInfo | undefined;
  owner: string;
  tab?: string;
}

const Pagination = (props: IProps) => {
  const navigate = useNavigate();

  const handlePreviousClick = () =>
    navigate(
      `/${props.owner}?before=${props.pageInfo?.startCursor || ''}${props.tab ? `&tab=${props.tab}` : ''
      }`
    );

  const handleNextClick = () =>
    navigate(
      `/${props.owner}?after=${props.pageInfo?.endCursor || ''}${props.tab ? `&tab=${props.tab}` : ''
      }`
    );

  return (
    <Show when={props.pageInfo}>
      <div class={styles.container}>
        <span class={styles.wrapper}>
          <button
            type="button"
            disabled={
              !props.pageInfo || !props.pageInfo.hasPreviousPage || !props.pageInfo.startCursor
            }
            onClick={handlePreviousClick}
            class={styles.btn_container + ' rounded-l-lg'}
          >
            Previous
          </button>
          <button
            type="button"
            onClick={handleNextClick}
            disabled={!props.pageInfo || !props.pageInfo.hasNextPage || !props.pageInfo.endCursor}
            class={styles.btn_container + ' rounded-r-lg'}
          >
            Next
          </button>
        </span>
      </div>
    </Show>
  );
};
```